tests/scripts/docker/black.sh

```diff
@@ -7,7 +7,7 @@ docker_pip_cache="/tmp/cache/pip"
 
 cd tests
 
-docker build --build-arg PYTHON_IMAGE=python:3.6 -t python-linters .
+docker build --build-arg PYTHON_IMAGE=python:3.8 -t python-linters .
```

Comment:
I wonder if these scripts are still in use. We use pre-commit for black/isort/flake8. The only reference I could find is in [`run-tests-locally.asciidoc`](https://github.com/elastic/apm-agent-python/blob/main/docs/run-tests-locally.asciidoc#code-linter)

From what I can tell, we might as well just delete these three scripts.

Reply:
Good point, let me review it in a follow up, as I don't wanna change any further behaviours in this PR but only bump the version to use a slim one and default to a major python version in those scripts.